step-release-vis/src/app/services/environmentService.ts

```diff
@@ -85,6 +85,40 @@ export class EnvironmentService {
     return polys;
   }
 
+  private formALine(a: Point, b: Point, c: Point): boolean {
+    return (
+      Math.abs(
+        a.x * b.y - b.x * a.y + b.x * c.y - c.x * b.y + c.x * a.y - a.x * c.y
+      ) < 0.0000001
```

Comment:
let's make this .1 (that is .5% of a pixel for the actual triangle, as the division is removed already)